scripts/bigshot.lic

```diff
@@ -2691,8 +2693,12 @@ class Bigshot
         next
       elsif result =~ /may not be activated within 60 seconds of a Multi-Strike\./i
         break
+      elsif result =~ /(?:Barrage|Flurry|Fury|Guardant Thrusts|Pummel|Thrash) is still in cooldown\./i
+        break
       elsif result =~ complete_regex || Time.now() > break_out
         break
+      elsif Effects::Buffs.active?("Assaulting")
+        break

```

Comment:
i think you want to not break on assaulting it would be more a safety like if cooldown is active but assault is not, then break (so you know you successful did the thing at some point, but you are not actively assaulting anymore)